.github/workflows/build-deploy-workflow.yml

```diff
@@ -0,0 +1,110 @@
+name: dan-plugin-arbt
+
+on:
+  push:
+    branches: [ main ]
+    paths-ignore:
+      - '**/README.md'
+      - '**/*.yml'
+  workflow_dispatch:
+
+env:
+  ROOT_SOLUTION_PATH: ./
+  FUNCTION_PROJECT_PATH: src/Altinn.Dan.Plugin.Arbeidstilsynet
+  AZURE_ARTIFACTS_FEED_URL: https://pkgs.dev.azure.com/nadobe/Nadobe/_packaging/Packages/nuget/v3/index.json
+  BUILD_CONFIGURATION: 'Release'
+
+  ARTIFACT_NAME: 'dan-plugin-arbt'
+  FUNCTIONAPP_NAME_TEST: 'func-esarbt-test-dev'
```

Comment:
Burde vi safe og holde navnene også som secrets? Det er kanskje like greit med tanke på det vi snakke om med composite actions også?

Reply:
Ulempen med secrets er at de er nettopp secrets, så du får ikke sett verdiene igjen etter at de er lagt inn. Kan kanskje gjøre det mindre lesbart/vanskeligere å debugge.
(Vi kan sikkert droppe ARTIFACT_NAME da dette bare er navnet på artefakten som bygges, og hentes opp igjen av deploysteget. Hvis vi ikke angir navn defaulter det til 'artifact'.)

Composite actions kan ta inn variabler, så kanskje greit at de forskjellige workflowene for komponentene sender disse navnene inn til felles composite actions?

Reply:
Siden de er "åpent tilgjengelig" på nett er det nok en fordel om de er secrets - og navngivingen er jo på standardformat så det burde ikke være noe problem :)

Reply:
Ok, du bestemmer :-)